src/linear_algebra/nonsingular_inverse.lean

```diff
@@ -403,12 +405,20 @@ B.nonsing_inv_left_right A h
 
 end inv
 
-lemma cramers_rule (A : matrix n n α) (b : n → α) (h : is_unit A.det) :
+/- One form of Cramer's rule. -/
+lemma mul_vec_cramer_of_det_is_unit (A : matrix n n α) (b : n → α) (h : is_unit A.det) :
```

Comment:
Whoops, sorry for that suggestion... this name doesn't really describe the statement.
```suggestion
lemma cramer_eq_det_smul_mul_vec_inv (A : matrix n n α) (b : n → α) (h : is_unit A.det) :
```
Or should we actually swap the two sides, and also make this a `simp` lemma?

Reply:
I should apologise; it was lazy of me to just copy and paste in that suggestion without thinking about it.

Yeah I like the idea of swapping the sides and making it `simp`. I've done that and attempted to apply a sensible name.